hack/functest.sh

```diff
@@ -15,7 +15,7 @@ echo "Running e2e test suite"
 export KUBECONFIG=$(./kubevirtci kubeconfig)
 export IMAGE_REPO=k8s.gcr.io
 export TENANT_CLUSTER_KUBERNETES_VERSION="v1.23.10"
-export NODE_VM_IMAGE_TEMPLATE=quay.io/capk/ubuntu-2004-container-disk:${TENANT_CLUSTER_KUBERNETES_VERSION}
```

Comment:
once CI passes, we need to remember to promote your image to the quay/capk repo and update this line before merging.